components/raftstore/src/store/config.rs

```diff
@@ -552,6 +555,7 @@ impl Default for Config {
             region_split_size: ReadableSize(0),
             clean_stale_peer_delay: ReadableDuration::minutes(0),
             inspect_interval: ReadableDuration::millis(100),
+            inspect_kvdb_interval: ReadableDuration::secs(0),
```

Comment:
Don't we want to set a proper default here? Like 2s?

Reply:
To disable it by default. Used to compatible with standard deployment scenarios where users deploy a TiKV node using a single disk.

A todo for it is that doing the check and override it with `2s` if using two different mount disks when initializing the configurations.